app/services/manifest_builder.rb

```diff
@@ -1,32 +1,61 @@
 class ManifestBuilder
-  attr_reader :record
+  attr_reader :record, :light
```

Comment:
Is the idea behind `:light` that if true only references to the Manifests are serialized?